.github/workflows/images.yaml

```diff
@@ -27,6 +27,11 @@ jobs:
         include:
           - name: cilium-cli
             dockerfile: ./Dockerfile
+            platforms: linux/amd64,linux/arm64
+          - name: cilium-cli-ci
+            dockerfile: ./Dockerfile
+            platforms: linux/amd64
+
```

Comment:
let's make this workflow change (pushing to quay.io/cilium/cilium-cli) in a separate pull request as a follow up. i (or @tklauser) needs to work with cilium maintainers to create new quay.io repo and such.

for this pull request, let's just push the multi arch image to quay.io/cilium/cilium-cli-ci.

Reply:
Am i wrong to think this is quite duplicate of what i was trying to achieve with #2755? we already have the option to build multiarch with the changes there